tasks/WaypointNavigator.cpp

```diff
@@ -58,9 +58,9 @@ void WaypointNavigator::updateHook()
         
         delta.time = base::Time::now();
         delta.linear = wp.cmd.linear - pose.position;
-        delta.angular(0) = wp.cmd.angular(0) - base::getRoll(pose.orientation);
-        delta.angular(1) = wp.cmd.angular(1) - base::getPitch(pose.orientation);
-        delta.angular(2) = wp.cmd.angular(2) - base::getYaw(pose.orientation);
+        delta.angular(0) = base::Angle::fromRad(wp.cmd.angular(0) - base::getRoll(pose.orientation)).getRad();
```

Comment:
In order to normalize the angular values you could also use the static function `base::Angle::normalizeRad`